extmod/network_wiznet5k.c

```diff
@@ -388,6 +379,17 @@ STATIC void wiznet5k_init(void) {
     };
     wiznet5k_obj.netinfo = netinfo;
 
+    // Configure wiznet provided TCP / socket interface
```

Comment:
This change makes sense (calling `ctlnetwork` after actually initialising netinfo), but maybe only the `ctlnetwork` line needs to move?

Reply:
Right, 
In conclusion, after initializing netinfo, call the ctlnetwork function and set the network-related register to the initialized value